relations/routes/tests/test_ui.py

```diff
@@ -0,0 +1,130 @@
+    def test_supercede(self, mock_supercede: Any) -> None:
+        """Endpoint /<string:arxiv_id_str>v<int:arxiv_ver>/relations/<string:relation_id_str> returns html about the newly created relation and the relation it superceded."""
+
+        post_data = {'resource_type':'type',
+                    'resource_id':1012,
+                    'description':'test relation',
+                    'creator':'tester'}
+        return_data={'arxiv_id':'TEST',
+                    'arxiv_ver':10,
+                    'relation_id':1012,
+                    'resource_type':'type',
+                    'description':'test relation'}        
+        mock_supercede.return_value = return_data, HTTPStatus.OK, {}
+        token = generate_token('1234', 'foo@user.com', 'foouser',
+                               scope=[])
+        response = self.client.post('/TESTv10/relations/relation1',
+                                    data=json.dumps(post_data),
+                                    headers={'Authorization':token},
+                                    content_type='application/json')
+
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+        self.assertEqual(response.headers['Content-Type'],
+                         'text/html; charset=utf-8')
+
+    @mock.patch(f'{ui.__name__}.controllers.suppress')
+    def test_suppress(self, mock_suppress: Any) -> None:
+        """Endpoint /<string:arxiv_id_str>v<int:arxiv_ver>/relations/<string:relation_id_str>/delete returns html about the newly created relation and the relation it suppressed."""
+
+        post_data = {'resource_type':'type',
+                    'resource_id':1012,
+                    'description':'test relation',
+                    'creator':'tester'}
+        return_data={'arxiv_id':'TEST',
+                    'arxiv_ver':10,
+                    'relation_id':1012,
+                    'resource_type':'type',
+                    'description':'test relation'}
+        mock_suppress.return_value = return_data, HTTPStatus.OK, {}
+        token = generate_token('1234', 'foo@user.com', 'foouser',
+                               scope=[])
+        response = self.client.post('/TESTv10/relations/relation1/delete',
+                                    data=json.dumps(post_data),
+                                    headers={'Authorization':token},
+                                    content_type='application/json')
+
+        expected_data = return_data
+
+        self.assertEqual(response.status_code, HTTPStatus.OK)
+        self.assertEqual(response.headers['Content-Type'],
+                         'text/html; charset=utf-8')
+
+    @mock.patch(f'{ui.__name__}.controllers.retrieve')
+    def test_get_relations(self, mock_retrieve: Any) -> None:
+        """Endpoint /<string:arxiv_id_str>v<int:arxiv_ver> returns html about all relations belonging to an id."""
+
+        return_data={'relations':[{'key1':'val1'},{'key1':'val2'}]}  
```

Comment:
Minor nit: spaces around ``=``, single space after ``,:``.

```suggestion
        return_data = {'relations': [{'key1': 'val1'}, {'key1': 'val2'}]}  
```